Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1091,7 +1091,7 @@
         SYS_thread_selfusage
         SYS_write))
 
-(deny syscall-unix (with telemetry))
+(deny syscall-unix)
```

Comment:
Should we just silence the `SYS_Persona` call? Or do we see many others we don't want to learn about?